cli_reference/basic_cli_operations.adoc

```diff
@@ -89,324 +88,324 @@ point the `oc` client to the correct API endpoint. For example: `oc get svc --ap
 == Basic CLI Operations
```

Comment:
I feel like now the "Object Types" section (which is currently at the very bottom of the topic) should maybe get slide up to here, between "Common Operations" and "Basic CLI Operations". Sets the understanding of the available objects types and their abbrevs, before moving on to the commands that actually operate on them.